app/assets/javascripts/components/common/calendar.jsx

```diff
@@ -151,6 +152,7 @@ const Calendar = createReactClass({
 
     const onWeekdayClick = this.props.editable ? this.selectWeekday : null;
     const onDayClick = this.props.editable ? this.selectDay : null;
+    const ariaModifier = this.props.useAriaLabel ? 'selected' : null;
```

Comment:
Why is it necessary to make changes outside of the WeekdayPicker component for this?

Reply:
Actually, I reviewed the code again and realized this change is not necessary. I have removed the existing `ariaModifier` since the `WeekdayPickerComponent` was not using it at all, and the Calendar component was not passing it either. I replaced it with the hardcoded string `selected`.

Additionally, I removed `aria-pressed` and replaced it with `aria-label`  & `aria-live`. 

Also, since the WeekdayPickerComponent is only used by the Calendar component, this change should not cause any issues.

Can you please review the updated code whenever you have time? Thank you 😃